agent/mission_solver/data_loader.py

```
@@ -0,0 +1,158 @@
+"""
+周常任务求解器 — 本地数据加载
+
+数据文件位于 assets/resource/Chaldea/，由 tools/update_quest_data.py 预下载。
```

Ruff (0.15.11)

[warning] Docstring contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF002)